CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+* Jameelah Young, Jarv.us (on behalf of Caltrans)
+* Elizabeth Sall, UrbanLabs LLC (on behalf of Caltrans)
+* Benjamin Bressette, Caltrans
+* Blake Fleisher, Jarv.us (on behalf of Caltrans)
+* Joey Reid, Metro Transit (Minneapolis-St. Paul, MN)
+
+PMT Group Member [GitHub Access](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization): Admin
+ 
+### Contributors
+
+Contributors actively work to develop the TIDES specification. They propose additions, modifications, and improvements to the speciation document through issues and pull requests in this GitHub repository.
+
+Contributors must request [here][contributor-registration] to be registered in order to gain access.  Requests to become a Contributor must be approved by project Leadership.
+ 
+Contributor Group Members: 
+The list of registered contributors is maintained in the [contributors.md][contributors-file] file.
+
+Contributor Group [GitHub Access](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization): Write
+ 
+### Stakeholders
+ 
+Stakeholders will be informed about progress on TIDES and given the opportunity to review the specification as it is developed.  They may provide comments on the specification by creating or responding to Issues in this repository.  Stakeholders are not able to generate or comment on pull requests.  To be included as a TIDES Stakeholder, join the TIDES Project Google Group.
+ 
+Stakeholder Group Membership:
+* Members of the TIDES Project Google Group 
+* Others who have expressed interest in following progress or contributing to TIDES, but who have not requested to be a registered Contributor
+ 
+Stakeholder Group [GitHub Access](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization): Read/Create Issues (same as general public)
+ 
+## [Review and Approval Process](review)
+ 
+Prior to release of Version 1.0 of the specification, the PMT and Leadership will have final approval of all changes.   All Contributors are permitted and encouraged to discuss and comment on issues and pull requests and make recommendations for changes to the specification.
+
+Leadership will convene a governance group to refine this and decide the approval process for Version 1.0 and the governance and approval process for future revisions to the specification.
+ 
+## [Code of Conduct](code-of-conduct)
+ 
+Contributors to the TIDES Project are expected to read and follow the Code of Conduct for the project in the [CODE_OF_CONDUCT.md][code-of-conduct-file] file.
+
+[contributor-registration]: https://forms.office.com/Pages/ResponsePage.aspx?id=i_a_3SpIc0WB4P74FWpP0Hpd6kyRp1VEg8rnx5-CwORUMFFGTzBYRktEMkJRWVg4Qlg3SkM0VEJKVi4u
+[license-file]: https://www.google.com
+[contributors-file]: https://www.google.com
+[code-of-conduct-file]: https://www.google.com
```

Comment:
Replace these with correct links, e.g., `[license-file]: LICENSE`, `[contributors-file]: CONTRIBUTORS.md`, and `[code-of-conduct-file]: CODE_OF_CONDUCT.md`.